java/client/src/main/java/glide/api/commands/SetBaseCommands.java

```diff
@@ -75,4 +75,24 @@ public interface SetBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> scard(String key);
+
+    /**
+     * Returns if <code>member</code> is a member of the set stored at <code>key</code>.
```

Comment:
This is poor wording.  We should consider updating this to: 
```
Returns <code>Boolean</code> indicating whether <code>member</code> is a member of the set stored at <code>key</code>.
```

Reply:
Was copied from Node and Py.